base/array.jl

```diff
@@ -1022,6 +1022,17 @@ function sizehint!(a::Vector, sz::Integer)
     a
 end
 
+"""
+    capacity(s)
+
+Returns the allocated buffer capacity of s.
+"""
+function capacity end
+
+function capacity(a::Vector)
+    return ccall(:jl_array_capacity, Csize_t, (Any,), a)
```

Comment:
This should return an `Int` in julia.

Reply:
@JeffBezanson, @StefanKarpinski  I'd be happy with either way of course. Please let me know which one is it!

Reply:
It's gotta be the latter since `jl_array_len` similar returns a `size_t` in C (and of course an `Int` in Julia).  But apparently I don't have permissions to commit Stefan's suggestion.